### PR TITLE
Trigger plugins/re on groups if regexbot is not in them

### PR DESCRIPTION
**Pull request by Lonami**

### CODE REVIEW

**Comment by Lonami** [comment]
Needless to say this won't work if there are more bots. I think getting the full channel returns *just* bots, which is really nice. But, have no time now ¯\\\_(ツ)\_/¯

File: plugins/re.py [hunk 1]
```diff
@@ -55,9 +55,16 @@ def actually_doit(original):
                 return m, s
         return None, None
 
+
```
Comment (9ary) [inline plugins/re.py]: Only one line between functions please

File: plugins/re.py [hunk 2]
```diff
@@ -72,10 +79,8 @@ def on_regex(event):
 
     raise events.StopPropagation
 
+
```
Comment (9ary) [inline plugins/re.py]: same

File: plugins/re.py [hunk 3]
```diff
@@ -55,9 +55,16 @@ def actually_doit(original):
                 return m, s
         return None, None
 
+
+def group_has_regex(group):
+    return any(getattr(x, 'username', None) == 'regexbot'
+               for x in client.get_participants(group, search='@regexbot'))
+
+
```
Comment (9ary) [inline plugins/re.py]: same

**Review by 9ary — COMMENTED** [review]
OwO
I'll wait till I have a message header to prevent people from using it against me though.

File: plugins/re.py (hunk 1 above)
Comment (Lonami) [reply plugins/re.py]: But PEP8.

Comment (9ary) [reply plugins/re.py]: PEP8 is not god.

**Comment by Lonami** [comment]
> I'll wait till I have a message header to prevent people from using it against me though.

What did you mean by this?

**Comment by 9ary** [comment]
I want to prefix messages with something like `[[regex]]` so that people won't be able to make me say something and then forward it as is.

**Comment by Lonami** [comment]
Nicely seen. Has it happened already? Maybe add a [**@mokubot**](https://t.me/mokubot)-style "*Did you mean:*"?

**Pull request merged**